src/components/Button/Button.tsx

```diff
@@ -8,7 +8,7 @@ import Spinner from '#components/Spinner/Spinner';
 
 type Color = 'default' | 'primary';
 
-type Variant = 'contained' | 'outlined' | 'text';
+type Variant = 'contained' | 'outlined' | 'text' | 'danger' | 'delete';
```

Comment:
Shouldn't `danger` and `delete` be a color instead of a button variant?

Reply:
I put them as variants as they additionally have different styles for the outlines (both of them) and `outlined` is already a variant, so having them as colors seemed like it could potentially overlap styles with each other and wasn't that intuitive to me.